packages/client/src/modules/user/modules/jwt/index.jsx

```diff
@@ -77,34 +91,56 @@ ProfileScreen.propTypes = {
   navigation: PropTypes.object
 };
 
-const LoginNavigator = StackNavigator({
-  Login: { screen: LoginScreen },
-  Profile: { screen: ProfileScreen }
-});
-
 export default new Feature({
   tabItem: {
-    User: {
-      screen: LoginNavigator,
+    Profile: {
+      screen: ProfileScreen,
+      userInfo: {
+        requiredLogin: true
```

Comment:
Why we cannot just rely on user roles or scopes? Don't you find this approach redundant?